oslo_vmware/rw_handles.py

```diff
@@ -310,30 +310,44 @@ def __init__(self, session, lease, url, file_handle,
 
     def _log_progress(self, progress):
         """Log data transfer progress."""
-        if (progress == 100 or (progress - self._last_logged_progress >=
-                                MIN_PROGRESS_DIFF_TO_LOG)):
-            LOG.debug("Data transfer progress is %d%%.", progress)
-            self._last_logged_progress = progress
+        if 'bytes_read' in progress:
+            metric = progress["bytes_read"]
+        else:
+            metric = progress["bytes_written"]
+
+        progress["metric"] = metric
+        time_diff = time.time() - self._last_progress_update
+        progress["time_diff"] = datetime.timedelta(seconds=time_diff)
+
+        LOG.info("Data Transfer progress: %(metric)s"
+                " of %(total_size)s in %(time_diff)s", progress)
```

Comment:
`time_diff` is nice, but not really necessary. You should be able to extract the time difference from the different log lines.

Reply:
When you have several backups going on it's hard to see what that difference is from the noise.  This just makes it easier IMHO